emantomo/convert/convert.py

```diff
@@ -29,6 +29,7 @@
 import glob
 import itertools
 import json
+import time
```

Comment:
I think this is not used. Probably was used to measure performance, correct?

Reply:
Below those lines there was an unused import, but time is used.